index.rst

```diff
@@ -179,68 +195,64 @@ Notes on "recommended" category
-top of the image list in the options form.
+prepulled, and in that it is only an alias to another tag.
+The thing that distinguishes the tag designated as "recommended" from all others is that it will always be prepulled, and always will be displayed at the top of the image list in the options form.
+In general, most users, most of the time, should use the image designated as "recommended" at whichever RSP instance they are using.
 
 Notes on alias tags
 ^^^^^^^^^^^^^^^^^^^
 
 An alias tag requires some special handling of the tag within
-cachemachine to determine what the image digest corresponding to the
+the Nublado controller to determine what the image digest corresponding to the
 recommended image is, and construct a mapping of its display name to the
 actual image--this information is used both in the spawner options form
 to tell the user what they're getting with and in the Lab UI to remind
 the user what they're using.
 
-Most users will use the "recommended" tag most of the time, and the
-tag is generally applied to the most recent weekly build, as soon as it
-has been possible to certify a weekly build by subjecting it to our QA
-process.
+Most users will use the "recommended" tag most of the time.
 
-Any other defined alias tags will appear in the list between "recommended"
-and the first of the non-alias images.
+Any other defined alias tags will appear in the Nublado spawner list between "recommended" and the first of the non-alias images.
 
 Release
 -------
 
-Release images are the (roughly twice a year) official stack releases.
-They are intended to be kept available indefinitely, and for the RSP
-machinery to, at any point in time, be able to run the current release
-and the two prior to it.  (If older releases are required, it may take
-some work, up to and including a separately-constructed RSP instance) to
-make them runnable.
+Release images are the official stack releases, historically on a roughly-twice-a-year-cadence.
+They are intended to be kept available indefinitely, and for the RSP machinery to, at any point in time, be able to run the current release and the two prior to it.
+If older releases are required, it may take some work, up to and including a separately-constructed RSP instance, to make them runnable.
+
+Historically, the "recommended" image during construction was usually a weekly; however, in operations, it will generally point to a release image.
 
 Tag Format
 ^^^^^^^^^^
 
 Release tags are now of the form ``r[major]_[minor]_[patch]``,
-e.g. ``r21_0_1``.  Prior to Release 18, they were not
-underscore-separated, e.g. ``r170``.  The first two digits are the major
-version, and the last one is the minor version.  In this form, the patch
-version is always 0.  Cycle and rest are permitted, so, for instance,
-all of ``r_21_0_1_c0019.001``, ``r_21_0_1_20210703``, and
-``r_21_0_1_c0019.001_20210703`` are allowed.
+e.g. ``r21_0_1``.
+Prior to Release 18, they were not underscore-separated, e.g. ``r170``.
```

Comment:
We don't care any more about the old format, right? I think we could drop all of that. So far as I know, we've deleted all of those images.

Reply:
Ah.  It still exists at least at Docker Hub, though.  We can probably drop the note from the documentation, but we can't remove it from the code.

Reply:
(I'm sure we've TRIED to reap it.  Docker Hub is really bad (or was last we tried, I think about a year ago) about not actually untagging/removing things you ask it to, while claiming it performed your request.